src/ApiProblem.php

```diff
@@ -243,7 +245,7 @@ public function setDetailIncludesStackTrace($flag)
      */
     protected function getDetail()
     {
-        if ($this->detail instanceof Throwable || $this->detail instanceof Exception) {
+        if ($this->detail instanceof Throwable) {
```

Comment:
since we support only 7.3 and above, remove redundant instanceof Exception conditions